### DISCUSSION TITLE
npm run dev is stucked

### DISCUSSION THREAD

**Discussion by 75310shubham** [vercel/next.js#52375] [Help]
### Summary

First I did
npx create-next-app@latest

Then installed all the relevant packages, 

At the time of npm run dev 
It is stucked. 

Using windows 10, visual code studio. 

Please help..

**Comment by icyJoseph** [comment]
This often happens if your node version is not correct.

`node -p 'process.arch'` run this command... does it match your system/OS architecture?

**Reply by 75310shubham** [reply]
**Thank you for your reply.**

`PS C:\Users\Shubham\Desktop\fun\home> node -p 'process.arch'
x64
PS C:\Users\Shubham\Desktop\fun\home> npm run dev

> home@0.1.0 dev
> next dev

- ready started server on 0.0.0.0:3000, url: http://localhost:3000
- event compiled client and server successfully in 1159 ms (20 modules)
- wait compiling...
- event compiled client and server successfully in 608 ms (20 modules)




`

It is stuck here, not doing anything

**Reply by icyJoseph** [reply]
And you are opening `localhost:3000` in your browser right? Otherwise it is just waiting for requests to come in.

The `x64` architecture does match your OS correct?

**Comment by mkercheval-tva** [comment]
I am have the same issue on Windows

NEXT.CONFIG: {
        "staticPageGenerationTimeout": 1000000,
        "images": {
                "loader": "custom"
        },
        "pageExtensions": [
                "page.tsx",
                "page.ts"
        ]
}

    Operating System:
      Platform: win32
      Arch: x64
      Version: Windows 10 Pro
    Binaries:
      Node: 18.16.1
      npm: N/A
      Yarn: N/A
      pnpm: N/A
    Relevant Packages:
      next: 13.4.9
      eslint-config-next: N/A
      react: 18.2.0
      react-dom: 18.2.0
      typescript: N/A
    Next.js Config:
      output: N/A

I have double checked node -p 'process.arch' and it matches.
